common/djangoapps/third_party_auth/migrations/0010_delete_historicalusersocialauth.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 3.2.13 on 2022-04-27 14:06
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('third_party_auth', '0009_historicalusersocialauth'),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
+            name='HistoricalUserSocialAuth',
+        ),
+    ]
```

Comment:
I think this migration might need to be added in a follow-up PR because of the way our deploys work.

Reply:
Wouldn't it cause migration sync tests to fail? Or do we need to temporarily disable those tests?

Reply:
Oh, yes, I just referred to the internal doc. Skipping the test in first step.